calcom/cal.com
# fix: Issue with zoom meetings getting "invalid Meeting ID"

## What does this PR do?



- Fixes #16601(GitHub issue number)
- Fixes CAL-[4319](https://linear.app/calcom/issue/CAL-4319/issue-with-zoom-meetings-getting-invalid-meeting-id)(Linear issue number - should be visible at the bottom of the GitHub issue description)



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Are there environment variables that should be set?
  - Ensure that the Zoom API credentials are correctly set up in the environment.
- What are the minimal test data to have?
  - Create a test appointment with a Zoom meeting.
  - Reschedule the appointment to trigger the update logic.
- What is expected (happy path) to have (input and output)?
  - The Zoom meeting ID should be correctly updated after rescheduling.
  - The updated meeting ID should be reflected in the appointment details.
- Any other important info that could help to test that PR
  - Ensure that the Zoom integration is enabled and properly configured in the Cal.com settings.
